packages/@okta/vuepress-site/docs/guides/submit-app/main/index.md

```diff
@@ -60,13 +60,48 @@ If the Okta OIN team identifies any issues in the initial review and QA testing
+
+The OIN Manager always shows the current status of your integration.
+
+#### Step 1: Initial Review
+
+* **Pending review by Okta**: The Okta OIN team is notified of your submission. Okta reviews the submission and notifies you by email when the submission review is complete.
+* **Action required**: Okta has reviewed your submission and found issues that require your attention. Check your email for results from the Okta initial review. Sign in to OIN Manager, update the requested details, and click **Submit for Review**. After the submission review is complete, the Okta OIN team moves it to Step 2 for QA testing.
+
+#### Step 2: Code Review
+
+* **Pending review by Okta**: The Okta OIN team is now conducting internal QA tests and notifies you by email when the QA review is complete. If the QA test is successful, your submission is published automatically in the OIN.
+* **Action required**: Okta has found QA issues that require your correction. Check your email for results from the Okta QA review. Make the requested changes as an update to your existing submission.
+* **Final review by Okta**: The Okta OIN team is conducting a final internal QA test based on previously requested changes and notifies you by email when the final QA review is complete. If the review is successful, your submission is published automatically in the OIN.
+
+#### Step 3: Published
+
+* Congratulations, your integration is published in the OIN!
+
+The following flowchart outlines the entire process:
+
+<div class="three-quarter">
+
+![ISV Submission process flow](/img/oin/isv-portal_submission_flow.png)
+
+</div>
+
 ### Submission support
 
 Getting your app integration in the OIN catalog involves two phases: creating a functional integration and submitting it through the OIN publication process. For each phase in the process, Okta has an associated support stream to assist you.
 
 When you are constructing your Okta integration, you can post a question on the [Okta Developer Forum](https://devforum.okta.com/).
 
-When you are troubleshooting a submission or need to know the current publication status, sign in to the [OIN Manager](https://oinmanager.okta.com) using your dev credentials. You can make any necessary changes to your submission there. If you have questions or need additional support to publish your app integration, you can reach out to the Okta OIN team directly at <oin@okta.com>.
+When you need help during the submission process, use the Support section in the App Integrations page after you sign in to the [OIN Manager](https://oinmanager.okta.com). This section provides you with the following resources from the [Okta developer portal](https://developer.okta.com/):
```

Comment:
"in the App Integrations page" > "of the App Integrations page"

"provides you with the following" > "provides the following"